packages/build/tests/core/snapshots/tests.mjs.md

```diff
@@ -2027,15 +2028,16 @@ Generated by [AVA](https://avajs.dev).
     ␊
     ␊
     > The following Node.js modules use dynamic expressions to include files:␊
+       - test␊
        - test_2␊
-       - @org/another-test␊
+       - @org/yet-another-test␊
     ␊
       Because files included with dynamic expressions aren't bundled with your serverless functions by default,␊
       this may result in an error when invoking a function. To resolve this error, you can mark these Node.js␊
       modules as external in the [functions] section of your \`netlify.toml\` configuration file:␊
     ␊
       [functions]␊
-        external_node_modules = ["test_2", "@org/another-test"]␊
+        external_node_modules = ["test", "test_2", "@org/yet-another-test"]␊
```

Comment:
Those test snapshots changes are expected. They are due to the fact that we switching some of the test fixtures from dynamic `require()` to dynamic `import()`. `esbuild` is detecting those differently, which leads in difference of behavior in those tests. However, the test is still indicating that the code works correctly.